chef_master/source/server_security.rst

```diff
@@ -310,6 +310,52 @@ These newer add-ons will also write all of their secrets to ``/etc/opscode/priva
 
 While the file does not contain passwords in plaintext, it is not safe to share with untrusted users. The format of the secrets file allows Chef Infra Server deployments to conform to regulations that forbid the appearance of sensitive data in plain text in configuration files; however, it does not make the file meaningfully more secure.
 
+SSL Encryption Between Chef Infra Server and External PostgreSQL
+================================================================
+
+**New in Chef Server X.Y:**  Chef Server X.Y introduces the capability to encrypt traffic between Chef Infra Server and an external PostgreSQL configuration over SSL.  These instructions assume familiarity with PostgreSQL administration, configuration, and troubleshooting.  To enable this encryption (typical scenario):
+
+#. Ensure that PostgreSQL has been built with SSL support compiled-in.  Consult the appropriate PostgreSQL documentation as necessary.
```

Comment:
is there a version recommendation? or a link to the documentation for a good supported version might be useful.

Reply:
Added.